crates/volta-core/src/monitor.rs

```diff
@@ -1,58 +1,84 @@
+use std::env;
 use std::io::Write;
+use std::path::PathBuf;
 use std::process::{Child, Stdio};
 
 use log::debug;
+use tempfile::NamedTempFile;
 
 use crate::command::create_command;
 use crate::event::Event;
 
-pub struct Monitor {
-    monitor_process: Option<Child>,
-}
-
-impl Monitor {
-    /// Returns the current monitor.
-    pub fn new(command: &str) -> Monitor {
-        Monitor {
-            monitor_process: spawn_process(command),
+/// Send event to the spawned command process
+// if hook command is not configured, this is not called
+pub fn send_events(command: &str, events: &[Event]) {
+    match serde_json::to_string_pretty(&events) {
+        Ok(events_json) => {
+            let tempfile_path = match env::var("VOLTA_WRITE_EVENTS_FILE") {
```

Comment:
Uber nit: I think elsewhere we've used `env::var_os` when checking for the existence of an environment variable.

Reply:
I've seen both in the code, but using `var_os` does make it easy to use `and_then` with this, so I updated that in fd3c6de8c6a532eefdcf45d11da727fc90d0cd89